packages/controller-utils/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.3]
+
+### Added
```

Comment:
Nit: I would call this a bugfix, so "Fixed" seems like a more appropriate category. That matches the version bump as well; we typically use a minor bump for features/enhancements, but a patch for bug fixes.